stable/concourse/README.md

```diff
@@ -303,20 +305,45 @@ postgresql:
 
 ### Credential Management
 
-By default, this chart will not use a [Credential Manager](https://concourse.ci/creds.html).
+Pipelines ususally need credentials to do things. Concourse supports the use of a [Credential Manager](https://concourse.ci/creds.html) so your pipelines can contain references to secrets instead of the actual secret values. You can't use more than one credential manager at a time.
+
+#### Kubernetes Secrets
+
+By default, this chart will use Kubernetes Secrets as a credential manager. For a given Concourse *team*, a pipeline will look for secrets in a namespace named `[namespacePrefix][teamName]`, e.g. `concourse-main`. The service account used by Concourse must have `get` access to secrets in that namespace. 
+
+Here are some examples of the lookup heuristics, given `credentialManager.kubernetes.namespacePrefix=concourse-`:
+
+In team `accounting-dev`, pipeline `my-app`; the expression `((api-key))` resolves to:
+
+1. the secret value in namespace: `concourse-accounting-dev` secret: `my-app.api-key`, key: `value`
+2. and if not found, is the value in namespace: `concourse-team-accounting-dev` secret: `api-key`, key: `value`
```

Comment:
Shouldn't this be: concourse-accounting-dev?

Reply:
thx, I'll correct that